velox/common/base/SpillStats.cpp

```diff
@@ -124,7 +124,7 @@ SpillStats SpillStats::operator-(const SpillStats& other) const {
   return result;
 }
 
-bool SpillStats::operator<(const SpillStats& other) const {
+std::strong_ordering SpillStats::operator<=>(const SpillStats& other) const {
```

Comment:
Why compare defined this way? I mean do we really need to check that all fields `>=` or `<=` other at the same time?

Reply:
This question is not actual here, because compare for SpillStats was removed